tests/boards/ehl_crb/gpio/src/main.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * @file
+ * @brief Sample Application for GPIO driver.
+ * This sample shows how to configure gpio pins to output direction to
+ * control pin levels, and also how to configure pins to input direction
+ * to trigger interrupts.
+ * The sample uses 2 pins, one as output pin, and the other
+ * as input pin. Connect these 2 pins, output pin will toggle the level,
+ * this would trigger interrupts for the input pin.
+ *
+ * @brief Hardware setup.
+ * Connect pins 13 and 17 of J1E2 header of EHL CRB board in loopback.
+ *
+ * @{
```

Comment:
This looks like a left-over copypaste perhaps?

Reply:
I reused an existing sample app, this format is not needed here, will remove.